web/components/widgets/info-tooltip.tsx

```diff
@@ -2,13 +2,28 @@ import { InformationCircleIcon } from '@heroicons/react/outline'
 import { Tooltip } from './tooltip'
 import clsx from 'clsx'
 
-export function InfoTooltip(props: { text: string; className?: string }) {
-  const { text, className } = props
+export function InfoTooltip(props: {
+  text: string
+  className?: string
+  children?: React.ReactNode
+}) {
+  const { text, className, children } = props
   return (
     <Tooltip className="inline-block" text={text}>
-      <InformationCircleIcon
-        className={clsx('text-ink-500 -mb-1 h-5 w-5', className)}
-      />
+      {children ? (
```

Comment:
Cool! Typically it is better to just have a different component rather than configure it with different modes based on options. Ok to ship this though.